src/Display/Tabs/VerticalTabs.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+
+import classNames from 'classnames';
+
+import TabPane, { Props as TabPaneProps } from './TabPane';
+
+import {
+  TabsContainer,
+  TabsHeader,
+  TabsBody,
+} from '../../Style/Display/TabsStyle';
+
+const VerticalTabs: Tabs = ({activeTab, onTabClick, children, className}) => {
```

Comment:
Yeah, even I thought that the code here should be optimized. But I read somewhere that node modules should be as precise as possible. Even if there are minute changes in two functions the functions should be written separately instead of being grouped, so that people can can understand and use them properly. Hence, didn't change it :sweat_smile: 
Will change this.

Reply:
Oh I see, that's interesting. Do you have the link to where you read it from? Would like to take a look if possible haha.

Reply:
I couldn't find it again. If I do I'll share the link.
But I just realised that neither two files nor a function like generateClass is require, because the class names which are changing right now are like `horizontal-tab`, or `horizontal-tabs-list` or `vertical-tab` so instead I've used `${alignment}-tabs-list` and `${alignment}-tab`. 
I guess that would be sufficient :sweat_smile: